src/MeshlessFV/MfvMusclSimulation.cpp

```diff
@@ -148,12 +143,13 @@ void MfvMusclSimulation<ndim>::MainLoop(void)
 
 
 
-  // Calculate terms due to self-gravity
-  if (mfv->self_gravity == 1) {
+  // Calculate terms due to self-gravity / stars
+  if (mfv->self_gravity == 1 || nbody->Nnbody > 0) {
```

Comment:
I added a swicth to the self-gravity function so that if there are stars present, but no self-gravity the function computes the star forces for the active particles, but not the self-gravity, so we do always get the force from the stars now. The reason I put the force from the stars in this function is that its called at the end of the timestep, whereas the hydro is done at the start of the step in the meshless.  We could instead have a seperate function to do the stars, but that would involve an extra loop over active particles so I thought is solution was best.